allofplos/corpus/plos_corpus.py

```diff
@@ -223,7 +219,7 @@ def repo_download(dois, tempdir, ignore_existing=True, plos_network=False):
         dois = set(dois) - set(existing_articles)
 
     for doi in tqdm(sorted(dois), disable=None):
-        url = URL_TMP.format(doi)
+        url = doi_to_url(doi)
```

Comment:
If we are not pulling from CREPO, I think we should not refer to plos_network anymore, if so, this is the time to delete every reference to this parameter (here and in other parts of the code)

Reply:
this is reasonable; I'll remove it everywhere.